src/Pipedrive.net/Clients/OAuthClient.cs

```diff
@@ -83,5 +83,40 @@ public async Task<OAuthToken> RefreshAccessToken(OAuthRefreshTokenRequest reques
             var response = await connection.Post<OAuthToken>(endPoint, body, "application/json", null, hostAddress).ConfigureAwait(false);
             return response.Body;
         }
+        
+        public async Task<OAuthToken> RefreshAccessToken(OAuthRefreshTokenRequest request)
```

Comment:
Duplicated `RefreshAccessToken`: it already exists

Reply:
Ok, found it https://pipedrive.readme.io/docs/app-uninstallation#section-token-revocation

Reply:
Fixed!